src/_gettsim/transfers/grunds_im_alter.py

```diff
@@ -10,7 +10,7 @@ def grunds_im_alter_m_eg(  # noqa: PLR0913
     unterhaltsvors_m_eg: float,
     grunds_im_alter_eink_m_eg: float,
     erwachsene_alle_rentner_hh: bool,
-    vermögen_bedürft_eg: float,
+    vermögen_bedürft_bg: float,
```

Comment:
Why is `bg` the correct grouping if the function is on `eg` level? Am I missing something?

Reply:
Ah sorry just saw your note. In this case, why don't we keep `vermögen_bedürft_eg` here and set `vermögen` as an individual-level input var and aggregate it to `eg`?

Reply:
You are right! That seems the better option and shouldn't be too much hassle. Will do

Reply:
Yeah, I think this will just fix #724 along the way. Thanks! Just to be sure, the individual-level input should be `vermögen_bedürft`, at least for now (at some point we could think of putting it together from individual pieces).

Reply:
I just opened #732 which introduces that change.